.github/workflows/publish-nym-contracts.yml

```diff
@@ -2,19 +2,18 @@ name: publish-nym-contracts
 on:
   workflow_dispatch:
   release:
-    types: [created]
+    types: [ created ]
 
 jobs:
   build:
     if: ${{ (startsWith(github.ref, 'refs/tags/nym-contracts-') && github.event_name == 'release') || github.event_name == 'workflow_dispatch' }}
-    runs-on: [self-hosted, custom-ubuntu-20.04]
+    runs-on: [ self-hosted, custom-ubuntu-20.04 ]
```

Comment:
This is a bit unexpected though, should be maybe use the GH hosted or ARC ubuntu runners here instead?

Reply:
good question - I don't know. for now I'd rather not change what's not needed : )